packages/slate-editor/src/components/OptionsGroup/OptionsGroup.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import type { OptionsGroupOption, ChooseOptionProps } from './Option';
+import { ChooseOption } from './Option';
+import styles from './OptionsGroup.module.scss';
+
+interface OptionsGroupCommon<T extends string> {
+    name: string;
+    options: OptionsGroupOption<T>[];
+    columns?: number;
```

Comment:
What do you think about adding a top-level `disabled` prop here to disable all options at once? Would be more convenient, and also we can keep the `options` array as a const and only dynamically calculate top-level `disabled` prop on the whole `OptionsGroup` parent.